### ISSUE TITLE
Website enhancements

### ISSUE THREAD

**Issue by rvosa** [naturalis/supersmart#31]
Note: this issue is assigned to @rvosa

The following enhancements need to be added to the website before we can go live:
* supporting organizations (e.g. NBC, ERC) need to be acknowledged with their logos
* the resources section needs to be expanded with relevant literature, software we use, project-level resources (mailing lists, wiki, bug tracker)
* a screencast of the installation process
* once `smrt` is available, a screencast of a simple run-through (contingent on #25)
* capitalize the words that go into SUPERSMART in the opening sentence
* clarify what "quits" means (i.e., return to the terminal)

**Comment by rvosa** [comment]
@diogoprov - late reply, but thanks! I am now going to close this issue as what is still outstanding is basically a duplicate of issue #44
